docs/proposals/202105256-cluster-class-and-managed-topologies.md

```diff
@@ -334,12 +334,22 @@ type LocalObjectTemplate struct {
   - `spec.topology.workers.machineDeployments[i].name` field must be unique within a Cluster
 
 - For object updates:
-  - If `spec.topology.class` is set, it cannot be unset or modified.
+  - If `spec.topology.class` is set, it cannot be unset and if it's unset it cannot be set.
```

Comment:
Q: are we going to enable rebase after this PR?
If not let's add
```suggestion
  - If `spec.topology.class` is set, it cannot be modified or unset and if it's unset it cannot be set.
```

Reply:
I think rebasing could feasibly be tested in the first run of integration testing for the clusterClass - we have the compatibility and reconciliation in place in the current version of the code, so I don't think including this is an issue at this point /cc @sbueringer

Reply:
I think Fabrizio's point is that the webhook currently forbids this and the proposal should be somewhat consistent with the code. So we should either make the code consistent with the proposal or the other way around.
(I forgot that the webhook currently blocks this when we talked about tests before)

Reply:
That makes sense - I've updated the proposal. We'll have to come back to this once we enable the behaviour in the webhook and have confidence in the code as it exists.